post/getting-started-preact/index.md

```diff
@@ -0,0 +1,322 @@
+    return (
+        <LocationProvider>
+            <Header />
+            <main>
+                <Router>
+                    <Route path="/" component={Home} />
+
+                    <Route path="/blog" component={BlogList} />
+                    <Route path="/404" component={NotFound} />
+                    <Route path="/blog/:slug" component={Blog} />
+                    <Route path="/:slug" component={Page} />
+                </Router>
+            </main>
+        </LocationProvider>
+    );
+}
+render(<App />, document.getElementById("app"));
+```
+
+## Creating Pages With Directus
+
+Create a new collection called pages - make the Primary ID Field a "Manually Entered String" called slug, which will correlate with the URL for the page. For example about will later correlate to the page localhost:3000/about.
+
+Create a text input field called title and a WYSIWYG input field called content. In Access Control, give the Public role read access to the new collection. Create 3 items in the new collection
+
+Inside of pages, create a new file called slug.jsx. This is a dynamic route, so a single file can be used for all of the top-level pages.
+
+```js
+import { useEffect, useState } from "preact/compat";
+import { useRoute, useLocation } from "preact-iso";
+import directus from "../utils/directus";
+import { readItem } from "@directus/sdk";
+export function Page() {
+    const { params } = useRoute();
+    const location = useLocation();
+    const [page, setPage] = useState({});
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        setLoading(true); // Set loading state to true when fetching data
+
+        const fetchData = async () => {
+            try {
+                const page = await directus.request(
+                    readItem("pages", params.slug)
+                );
+                setPage(page); // Assuming posts.data contains an array of blog posts
+            } catch (error) {
+                location.route("/404");
+                console.error("Error fetching page:", error);
+            } finally {
+                setLoading(false);
+            }
+        };
+
+        fetchData();
+    }, [params.slug]);
+
+    return (
+        <div>
+            {loading ? (
+                <p>Loading...</p>
+            ) : (
+                <>
+                    <h1>{page.title}</h1>
+                    <div dangerouslySetInnerHTML={{ __html: page.content }} />
+                </>
+            )}
+        </div>
+    );
+}
+```
+
+Go to http://localhost:3000/about, replacing about with any of your item slugs. Using the Directus JavaScript SDK, the single item with that slug is retrieved, and the page should show your data. readItem() only checks against your slug Primary ID Field.
+
+![About Page](./images/about-page.png)
+Note that we check if a returned value exists, and return a 404 if not. Please also note thatv-html should only be used for trusted content.
```

Comment:
```suggestion
Note that we check if a returned value exists, and return a 404 if not. Please also note that `dangerouslySetInnerHTML ` should only be used for trusted content.
```

I'm going to stop reviewing here because this is a clear copy and paste without a check-through.